lib/utils.js

```diff
@@ -29,6 +29,9 @@ function configureBinaryLog (opts) {
     let l = this.levels[level];
     if (l < this.levels[this.level]) return; // eslint-disable-line curly
     actualLog(level, prefix, msg);
+
+    // TODO: Callback function to handle logs
```

Comment:
what is the purpose of the TODO comment?

Reply:
OK, I will remove it.

Reply:
hm. the comment is still there

Reply:
still there, unfortunately. `// TODO: Callback function to handle logs` @zcmgyu

Reply:
Sorry, updated in the latest commit.

Reply:
np, thanks for your help!